src/git.js

```diff
@@ -96,13 +97,11 @@ class Git {
 		let email = GIT_EMAIL
 
 		if (email === undefined) {
-			if (IS_INSTALLATION_TOKEN) {
-				core.setFailed('When using an installation token you must provide GIT_EMAIL and GIT_USERNAME')
-				process.exit(1)
+			if (!IS_INSTALLATION_TOKEN) {
+				const { data } = await this.github.users.getAuthenticated()
+				email = data.email
+				username = data.login
```

Comment:
This change was to disable the authentication in the git client when the token is being used